lumigator/python/mzai/lm_buddy/BUILD.pants

```diff
@@ -0,0 +1,11 @@
+python_sources(
```

Comment:
Adding a build file so lm buddy will build with the rest of the platform, but not compile to a PEX